sa/model.go

```diff
@@ -617,3 +617,16 @@ type keyHashModel struct {
 	CertNotAfter time.Time
 	CertSerial   string
 }
+
+var stringToSourceInt = map[string]int{
+	"API":           0,
```

Comment:
Let's leave 0 as unused, so it's obvious if we left the value unset somewhere. We can use "1" and "2".